pyrandwalk/pyrandwalk_obj.py

```diff
@@ -84,20 +84,30 @@ def prob_sec(self, sequence, initial_dist=None):
             current_state = next_state
         return probability
 
-    def run(self, ntimes=10, show=False):
+    def run(self, ntimes:int=10, show:bool=False, starting_state:int=None):
```

Comment:
We will handle Python typings in another soon-coming issue.
Please change it to:
```
def run(self, ntimes=10, show=False, starting_state=None):
```

Reply:
Ok sounds good! Will fix this